src/common/types/key.type.ts

```diff
@@ -0,0 +1,4 @@
+export enum Key {
+    Enter = 'Enter',
+    Escape = 'Escape',
+}
```

Comment:
Is there really a need for this?

Reply:
I just wanted to add this to enable typing here, it's opt-in though